setup/smart_home_data.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE IF NOT EXISTS "iot"."smart_home_data" (
+   "time" INTEGER,
+   "use_kw" DOUBLE PRECISION,
+   "gen_kw" DOUBLE PRECISION,
+   "house_overall" DOUBLE PRECISION,
+   "dishwasher" DOUBLE PRECISION,
+   "home_office" DOUBLE PRECISION,
+   "fridge" DOUBLE PRECISION,
+   "wine_cellar" DOUBLE PRECISION,
+   "garage_door" DOUBLE PRECISION,
+   "kitchen" DOUBLE PRECISION,
+   "barn" DOUBLE PRECISION,
+   "well" DOUBLE PRECISION,
+   "microwave" DOUBLE PRECISION,
+   "living_room" DOUBLE PRECISION,
+   "temperature" DOUBLE PRECISION,
+   "humidity" DOUBLE PRECISION
+)
```

Comment:
```suggestion
);
```
In case someone pastes the two statements into an SQL editor and tries to run them, semicolons will be needed.